recipes/full_finetune_single_device.py

```diff
@@ -344,20 +392,28 @@ def train(self) -> None:
                     self._metric_logger.log_dict(
                         {
                             "loss": loss.item(),
-                            "lr": self._optimizer.param_groups[0]["lr"],
+                            # NOTE: for optim in backward, this assumes all optimizers have the same LR. This is currently
+                            # true since we don't expose the ability to configure this yet.
+                            "lr": list(self._optim_ckpt_wrapper.optim_map.values())[
```

Comment:
can you make a method or property in the wrapper that retrieves this so this can be cleaner?